contents/handbook/growth/marketing/index.md

```diff
@@ -37,18 +37,220 @@ To incentivise YC startups to choose PostHog for the product analytics, we'll of
+  - Classify existing content into themes and rank effectiveness
+  - Experiment within popular and new (e.g case studies) content themes
+  - Create a content schedule
+- Community
+  - Make it easier to contribute to in-the-moment and long-running discussions using open tools
+  - Define areas community members can get involved
+    - Create getting started content for each area and different experience levels
+- Metrics:
+  - Implement remaining dashboards except more complex custom queries like "popular content"
+  - Define growth goals for Q3
+- Campaign: Improved YC deal for S21
+  - Continue to deliver and report on campaign
+
+### Q3
+
+- Product marketing
+  - Continue experiments
+  - Create product marketing for new features
+  - Complete content for remaining new personas
+- Content marketing
+  - Conduct design experiments to improve engagement and conversion
+  - Update themes
+  - Continue to experiment
+- Paid:
+  - Conduct various paid experiments
+- Campaign: Community events
+- Campaign: Community member profile of contributions and achievements
+- Campaign: Enterprise early adopters
+  - Conduct early marketing experiments
+- Campaign: Improved YC deal for S21
+  - Complete and report on campaign
+  - Plan and deliver post batch campaign
+- Metrics:
+  - Implement dashboards for complex custom queries
+- Hiring: Assess hiring needs in most likely areas:
+  - product, content, community
+
+### Q4
+
+- Product marketing
+  - Continue experiments
+  - Create product marketing for new features
+  - Update research and personas, expanding into enterprise
+  - Start creating enterprise product marketing
+- Content marketing
+  - Create specialization areas for team (e.g startups, enterprise, developers, non-technical)
+  - Update themes
+  - Continue to experiment
+- Paid:
+  - Expand experiments
+  - Set budget and targets for Q1 2022
+- Ops:
+  - Identify ops bottlenecks
+- Campaign: Enterprise early adopters
+  - Expand experiments
+- Campaign: Plan YC W22
+- Metrics:
+  - Report on past year
+  - Define growth goals for 2022
+- Hiring:
+  - continue to assess hiring needs in most likely areas:
+    - If not fulfilled: product, content, community
+    - paid, ops
+  - Set targets and discuss budget for 2022
+
+## Metrics
+
+### Acquisition
+
+Marketing wholly defines the Acquisition/Awareness metrics, and indirectly invested in the remaining categories of AARRR.
+
+In 2021 we'll build our comprehensive metrics suite, and identify the things that have the most impact throughout the funnel and not just at obvious conversion event.
+
+For year, month, week, day:
+
+- Conversions: Downloads, cloud signups, sales contacts, newsletter subs, community signups:
+  - Track event
+  - Total and rate (relative to page views, sessions, uniques)
+  - List of top x pages amongst conversions
+  - List of top x first-visit pages amongst conversions
+  - List of top x referral sources amongst conversions
+
+- Website: /, docs, handbook:
```

Comment:
I think it would be worth considering conversion in terms of activated users (or even tie it a bit further downstream to retention) because ultimately we want to make sure we're not only bringing users but bringing the "right" users to the product. Thoughts? CC @kpthatsme

Reply:
I feel like this is a pretty good starting point for us. 

One concern with tying it to activation is that our product is still evolving, and we're still understanding what our users want. So a dip in activation could be reflective in us not bringing in the "right" users, or it could be that our product isn't doing a great job of matching what acquired users actually want to accomplish.

I think perhaps a good starting point here would be to measure acquisition to the point of completing team creation and we can tie it into deeper metrics as our learnings grow in that area.

Reply:
I see conversion events for aquisition/awareness as a measurement of marketing and once signed-up there are a series events for different types of users that signify the activation event. 

## Example events that indicate activation:
### Solol dev user:
- add event tracking
- log in 2 times a week for 2 weeks
- create a dashboard
- other dev orientated indicators
- over x time spent on platform 

### Dev team:
- Invites team members
- % of team members pass individual activation

Reply:
Definitely agree on the evolving activation metric, and we do need to figure that out first. However, I do think we need some sort of downstream metric for marketing here. If we only measure conversion to team creation, we're not measuring that we're attracting the "right users" (e.g. getting a non-technical enthusiast to sign up wouldn't really be helpful), but open to push back here.

Reply:
Thanks Paolo, yes I agree we do need this. I believe Kunar is working on it this sprint. Also for now our marketing will be focussing on personas we do know retain.